generic_packer_template.json.j2

```diff
@@ -3,16 +3,16 @@
         {% if 'artefact_bucket' in event %}
         "artefact_bucket": "{{ event['artefact_bucket'] }}",
         {% endif %}
-        "http_proxy": "{% raw %}{{ env `HTTP_PROXY` }}{% endraw %}",
-        "https_proxy": "{% raw %}{{ env `HTTPS_PROXY` }}{% endraw %}",
+        "http_proxy": "{% raw %}{{ env `http_proxy` }}{% endraw %}",
+        "https_proxy": "{% raw %}{{ env `https_proxy` }}{% endraw %}",
         "no_proxy": "{% raw %}{{ env `NO_PROXY` }}{% endraw %}"
     },
     "builders": [{
       "type": "amazon-ebs",
       "source_ami_filter": {
         "filters": {
           "virtualization-type": "{{ event['source_ami_virt_type'] or 'hvm' }}",
-          "name": "{{ event['source_ami_name'] or 'amzn2-ami-hvm-*' }}",
+          "name": "{{ event['source_ami_name'] or 'amzn-ami-hvm-*' }}",
```

Comment:
Why has this changed back to an AL1 base AMI?

Reply:
it should not have been. Fixed.